src/components/content-tab-snapshots.tsx

```diff
@@ -151,6 +152,13 @@ function SnapshotRow( {
 						isSnapshotLoading={ snapshot.isLoading }
 						tagline={ __( "We're creating your new demo site." ) }
 					/>
+					<Button
+						aria-description={ __( 'Clear expired site' ) }
```

Comment:
This won't be needed as screen readers take the description based on the button's label, which in this case is the same string "Clear expired site".

Reply:
Updated in https://github.com/Automattic/studio/pull/218/commits/10e71e3075aff4dec6279fc102d08e4d51802a4f